examples/libp2p-in-the-browser/README.md

```diff
@@ -11,23 +11,6 @@ cd ./examples/libp2p-in-the-browser
 npm install
 ```
 
-## Signaling Server
-
-This example uses the `libp2p-webrtc-star` module, which enables libp2p browser nodes to establish direct connections to one another via a central signaling server. For this example, we are using the signaling server that ships with `libp2p-webrtc-star`.
-
-You can start the server by running `npm run server`. This will start a signaling server locally on port `9090`. If you'd like to run a signaling server outside of this example, you can see instructions on how to do so in the [`libp2p-webrtc-star` README](https://github.com/libp2p/js-libp2p-webrtc-star).
-
-When you run the server, you should see output that looks something like this:
-
-```log
-$ npm run server
-
-> libp2p-in-browser@1.0.0 server
-> star-signal
-
-Listening on: http://0.0.0.0:9090
-```
-
 ## Running the examples
 
 Once you have started the signaling server, you can run the Parcel server.
```

Comment:
Can you do a quick pass to make sure we're not referencing the local signaling server? We could do a "optionally run the signaling server locally" section, otherwise I would just get rid of this and clean up the script for running the signaling server to avoid confusion